app/controllers/usersController.js

```diff
@@ -1,24 +1,14 @@
 import { users } from '../models/users';
+import client from '../config/database';
 const appSecreteKey = 'hckjdsjsdadnbqdkjdqxbjkqwkn'
 import jwt from 'jsonwebtoken';
 export class userController{
 signUp(req, res){
-    const user = {
-        id: users.length + 1,
-        email: req.body.email,
-        firstName: req.body.firstName,
-        lastName: req.body.lastName,
-        password: req.body.password,
-        phoneNumber: req.body.phoneNumber,
-        address: req.body.address,
-        isadmin: req.body.isadmin
-    };
-    
+    const userQuery = 'INSERT INTO users(firstName,lastName,email,address,phoneNumber,password,isadmin) VALUES($1,$2,$3,$4,$5,$6,$7) RETURNING *';
```

Comment:
Please do this in a separate file called services. Your controller shouldn't do a lot of work and it shouldn't interact directly with the database. e.g. userService.js -> This file should contain methods that interact with the user model to create, read, update or delete data.